docs/source/getting-started/installation.rst

```diff
@@ -3,227 +3,195 @@
-          # Nebius is only supported for Python >= 3.10
-          pip install "skypilot[nebius]"
-          # Clouds below are supported/maintained by community/cloud providers.
-          pip install "skypilot[oci]"
-          pip install "skypilot[lambda]"
-          pip install "skypilot[runpod]"
-          pip install "skypilot[fluidstack]"
-          pip install "skypilot[paperspace]"
-          pip install "skypilot[cudo]"
-          pip install "skypilot[shadeform]"
-          # IBM is only supported for Python <= 3.11
-          pip install "skypilot[ibm]"
-          # SCP is only supported for Python <= 3.11
-          pip install "skypilot[scp]"
-          pip install "skypilot[vsphere]"
-          # Seeweb is only supported for Python >= 3.10
-          pip install "skypilot[seeweb]"
-          pip install "skypilot[primeintellect]"
-
-          pip install "skypilot[all]"
-
-
-    .. tab-item:: Nightly
-        :sync: nightly-tab
+      # install dependencies for the clouds you want to use
+      pip install "skypilot[kubernetes,aws,gcp]"
+  
+  .. tab-item:: uv venv
+    :sync: uv-venv-tab
 
-        .. code-block:: shell
+    .. code-block:: shell
 
-          # Recommended: use a new conda env to avoid package conflicts.
-          # SkyPilot requires 3.7 <= python <= 3.13.
-          conda create -y -n sky python=3.10
-          conda activate sky
-
-          # Choose your infra:
-
-          pip install "skypilot-nightly[kubernetes]"
-          pip install "skypilot-nightly[aws]"
-          pip install "skypilot-nightly[gcp]"
-          pip install "skypilot-nightly[azure]"
-          pip install "skypilot-nightly[coreweave]"
-          # Nebius is only supported for Python >= 3.10
-          pip install "skypilot-nightly[nebius]"
-          # Clouds below are supported/maintained by community/cloud providers.
-          pip install "skypilot-nightly[oci]"
-          pip install "skypilot-nightly[lambda]"
-          pip install "skypilot-nightly[runpod]"
-          pip install "skypilot-nightly[fluidstack]"
-          pip install "skypilot-nightly[paperspace]"
-          pip install "skypilot-nightly[do]"
-          pip install "skypilot-nightly[cudo]"
-          pip install "skypilot-nightly[shadeform]"
-          pip install "skypilot-nightly[ibm]"
-          pip install "skypilot-nightly[scp]"
-          pip install "skypilot-nightly[vsphere]"
-          # Seeweb is only supported for Python >= 3.10
-          pip install "skypilot[seeweb]"
-          pip install "skypilot-nightly[primeintellect]"
-
-          pip install "skypilot-nightly[all]"
-
-
-    .. tab-item:: From Source
-        :sync: from-source-tab
-
-        .. code-block:: shell
+      # Create a virtual environment with pip pre-installed (required for SkyPilot)
+      # SkyPilot requires 3.7 <= python <= 3.13.
+      uv venv --seed --python 3.10
+      source .venv/bin/activate  # On Windows: .venv\Scripts\activate
```

Comment:
For windows, we should suggest `WSL` instead of using the windows path, as the latter is not fully tested.